README.md

```diff
@@ -320,7 +320,56 @@ for (const conversation of myAppConversations) {
 }
 ```
 
-#### Handle different types of content
+### Send a broadcast message
+
+You can send a broadcast message (1:many message or announcement) with XMTP. The recipient sees the message as a DM from the sending wallet address.
+
+1. Use the bulk query `canMessage` method to identify the wallet addresses that are activated on the XMTP network.
+2. Send the message to all of the activated wallet addresses.
+
+For example:
+
+```js
+const ethers = require('ethers')
+const { Client } = require('@xmtp/xmtp-js')
+
+//Rate limits
+//1k publish request /5 minutes / IP
+//publish request would be Client.create() , send() and newConversation()
+//10k of any request type / 5 mins / IP
+
+async function main() {
+  //Create a random wallet for example purposes. On the frontend you should replace it with the user's wallet (metamask, rainbow, etc)
+  const wallet = ethers.Wallet.createRandom()
+  //Initialize the xmtp client
+  const xmtp = await Client.create(wallet)
+  console.log('Broadcasting from: ', xmtp.address)
+
+  //In this example we are going to broadcast to the GM_BOT wallet (already activated) and a random wallet (not activated)
+  const GM_BOT = '0x937C0d4a6294cdfa575de17382c7076b579DC176'
+  const test = ethers.Wallet.createRandom()
+  const broadcasts_array = [GM_BOT, test.address]
+
+  //Querying the activation status of the wallets
+  const broadcasts_canMessage = await Client.canMessage(broadcasts_array)
+  for (let i = 0; i < broadcasts_array.length; i++) {
+    //Checking the activation status of each wallet
+    const wallet = broadcasts_array[i]
+    const canMessage = broadcasts_canMessage[i]
+    console.log(wallet, canMessage)
+    if (broadcasts_canMessage[i]) {
+      //If activated, start
+      const conversation = await xmtp.conversations.newConversation(wallet)
+      // Send a message
+      const sent = await conversation.send('gm')
+      console.log(sent.sent)
```

Comment:
for this example code, WDYT of removing these `console.log` calls?

Reply:
Removed the console.log calls - ty!